### DISCUSSION TITLE
How do you get the timezone name?

### DISCUSSION THREAD

**Discussion by pzelnip** [starship/starship#2360] [Q&A] [answered]
The [chrono docs](https://docs.rs/chrono/0.4.7/chrono/format/strftime/index.html) seem to day that `%Z` will give the name (ex: "PST"), but when I do the following format on the time module:

```
[time]
disabled = false
format = '[\[$time\]](green) '
time_format = "%a %b %d %Y %l:%M%p (%Z%z)"
```
I see:

```
[Sun Feb 21 2021  1:44PM (-08:00-0800)]
```
Which seems like the `%Z` is resolving to `-08:00` instead of `PST`.  🤔

**Comment by chipbuster** [comment, marked as answer]
Sadly, this looks like an [upstream issue in chrono](https://github.com/chronotope/chrono/issues/288) that is not so easy to solve. In fact, the relevant test for this feature is already commented out in chrono source code, but the docs haven't been updated to match.

**Reply by pzelnip** [reply]
Hrm, that's annoying, though understandable.  Thanks!.

**Comment by pzelnip** [comment]
My current workaround was do use a custom command:

```
[custom.tztime]
command = 'date +"%a %b %d %Y %l:%M%p (%Z%z)"'
when = "true"
format = '[\[$symbol($output)\]](green)'
```
